src_c/font.c

```diff
@@ -762,9 +764,15 @@ static void
 font_dealloc(PyFontObject *self)
 {
     TTF_Font *font = PyFont_AsFont(self);
-
-    if (font && font_initialized)
+    if (font && font_initialized) {
+        if (self->ttf_init_generation != current_ttf_generation) {
+            // Since TTF_Font is a private structure
+            // it's impossible to access face field in a common way.
+            int** face_pp = font;
```

Comment:
I wonder if this is enough to stop the problem instead?
```c
TTF_CloseFont(font);
font=NULL;
```

Reply:
Because of tp_free we are able not to aware about font. Therefore setting font to NULL is just redundant. Also font_dealloc is called only once after TTF_Quit to get a segfault inside a function TTF_CloseFont.

Reply:
Ok, thanks for the explanation :)